src/main/java/com/zendesk/maxwell/producer/RabbitmqProducer.java

```diff
@@ -26,7 +26,7 @@ public RabbitmqProducer(MaxwellContext context) {
 		factory.setVirtualHost(context.getConfig().rabbitmqVirtualHost);
 		try {
 			this.channel = factory.newConnection().createChannel();
-			this.channel.exchangeDeclare(exchangeName, context.getConfig().rabbitmqExchangeType, context.getConfig().rabbitMqExchangeDurable);
+			this.channel.exchangeDeclare(exchangeName, context.getConfig().rabbitmqExchangeType, context.getConfig().rabbitMqExchangeDurable, context.getConfig().rabbitMqExchangeAutoDelete, null);
```

Comment:
Have you checked if `null` is safe here? Should we use an empty `Map`?

Reply:
The classes ChannelN and AutorecoveringChannel have several overloaded variants of this method exchangeDeclare. Before my change the arguments were not given. In case the map was not given as parameter, they always call another overload with null. Also when you check AMQImpl, there is following snippet:

 this.arguments = arguments==null ? null : Collections.unmodifiableMap(new HashMap<String,Object>(arguments));

To me it seems quite safe to use null here.